packages/utils/telemetry-utils/src/telemetryEventBatcher.ts

```diff
@@ -116,24 +121,25 @@ export class TelemetryEventBatcher<TMetrics extends string> {
 			...this.eventBase,
 		};
 
+		telemetryEvent.avgcAccumulatedDuration = this.accumulatedDuration /= this.counter;
```

Comment:
I think we want this to be the duration property of the final event we log. I thought you'd get a compile-time error on line 138, complaining that `telemetryEvent` is not an `ITelemetryPerformanceExt` because we can't guarantee it has a `duration` property, but I realized the duration property is optional. Still, that's in performance logs we're going to be looking for that property name, so I think this makes sense.
```suggestion
		telemetryEvent.duration= this.accumulatedDuration /= this.counter;
```